test_vm/Vagrantfile

```diff
@@ -16,16 +19,32 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.omnibus.chef_version = :latest
   config.vbguest.auto_update = true
 
-  config.vm.box = "centos/7"
-  config.vm.box_url = "centos/7"
+  #config.vm.box = "centos/7"
+  #config.vm.box_url = "centos/7"
+  config.vm.box = "centos7-base"
+  config.vm.box_url = "centos7-base"
   config.vm.host_name = "dbfitvm"
   config.vm.network "private_network", ip: "192.168.33.10"
-  config.vm.network "forwarded_port", guest: 8085, host: 8085
+  # dbfit(8085), oracle(1521), postgresql(5432), netezza(5480), informix(16466), mysql(3306), sqlserver(1433), db2(50000)
+  %w{8085 1521 5432 5480 16466 3306 1433 50000}.each do |ip|
+    config.vm.network "forwarded_port", guest_ip: '127.0.0.1', host_ip: '127.0.0.1', guest: ip, host: ip 
+  end
+  
   config.vm.synced_folder "..", "/var/dbfit", CONFIG['synced_folder_options']
 
   config.vm.provision :chef_solo do |chef|
-    chef.add_recipe "dbfit_test"
-
+    chef.add_recipe("dbfit_test")
+    chef.arguments = "--chef-license accept"
+    chef.run_list = ["recipe[firewalld::disable]", "recipe[iptables::disabled]",
```

Comment:
Do we need to explicitly list recipes? dbfit_test `default.rb` already includes list of included recipes..., is it failing to do the job?

Reply:
I agree with you this looks ugly, but I've had strange chef errors about missing cookbooks. Later I realized that this was due to some cookbooks that seem to be inkompatibel meanwhile an announced to be deprecated. I've more experience with puppet and this is the first time I've used chef so I'll try to revert this part to the original one. It is redundant to the includes...

Reply:
I have reverted the enumeration in the runlist and it runs ok now.